machineconfiguration/v1/zz_generated.crd-manifests/0000_80_machine-config_01_controllerconfigs-CustomNoUpgrade.crd.yaml

```diff
@@ -669,7 +669,7 @@ spec:
                                             when type is Name, and forbidden otherwise
                                           rule: 'has(self.type) && self.type == ''Name''
                                             ?  has(self.name) : !has(self.name)'
-                                      maxItems: 32
+                                      maxItems: 1
```

Comment:
This seems to be unrelated?

Reply:
This and the deletion of `DualReplica` keeps showing up every time I run `make update`.

Reply:
Make sure you rebase onto the latest main before you run the update command